src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -716,7 +716,7 @@ ORDER BY
     logicalrelid,
     shardmaxvalue::integer,
     shardid,
-    placementid;
+    nodeport;
```

Comment:
I don't have strong objections but just wanted to understand more. How could this change help for pg 12 compatibility? 

If the output of the test is not consistent across pg versions, I cannot see how this would fix it.

Reply:
The projection doesn't use `placementid` so the ordering had ties